install.sh

```diff
@@ -167,5 +167,55 @@ runchecksonstart: true
 loglevel: error
 EOF
 
-printSuccess "The Site Wide Analysis Tool Agent has been successfully installed at $agentPath"
-[ "$installDaemon" ] && installAndConfigureDaemon || installAndConfigureCron
+echo "** Install validation "
+
+if [[ ! -f "$appRoot/app/etc/env.php" ]]; then
+  error_exit "Magento not found."
+else
+  echo "Magento Found - OK"
+fi
+
+if nc -z $updaterDomain 443 2>/dev/null; then
+    echo "Connect to API Server - OK"
+else
+    error_exit "Can not connect to API Server"
+fi
+
+if [ -f "$agentPath/config.yaml" ]; then
+    echo "Config File is created - OK"
+else
+    error_exit "Config File was not created."
+fi
+
+phpVersion=$($phpPath -v | awk '{ print $2 }' | head -1)
+semver=( ${phpVersion//./ } )
+major="${semver[0]}"
+minor="${semver[1]}"
+
+echo "**Checking php version."
+
+if [ "$major" -eq 7 ]; then
+    if [ "$minor" -gt 2 ]; then
+        echo "php version - OK"
+    else
+        echo "You can specify another phpPath using env AGENT_INSTALLER_PHP."
+        error_exit "php engine reachable by $phpPath is $phpVersion and is not supported."
+    fi
+else
+    echo "php version - OK"
+fi
+
+mkdir $agentPath/tmp
+
+if [ -w "$agentPath/tmp" ] ; then
+  echo "Temporary Folder is writeable - OK"
+else
+  error_exit "Temporary Folder in agent directory is not writable"
+fi
+firstRun=$("$agentPath/scheduler")
+if [[ "$firstRun" == *"is going to update"* ]]; then
+  printSuccess "The Site Wide Analysis Tool Agent has been successfully installed at $agentPath"
+  [ "$installDaemon" ] && installAndConfigureDaemon || installAndConfigureCron
+else
+  error_exit "Failed to update launcher"
```

Comment:
```
error_exit "Failed to update launcher"
```
Please give a more guidance, what needs to be done next?
 - Start over again from the scratch
 - Check the logs